jetty-core/jetty-http/src/main/java/org/eclipse/jetty/http/HttpFields.java

```diff
@@ -536,15 +790,15 @@ default boolean isEqualTo(HttpFields that)
      *
      * <p>The cookie handling provided by this class is guided by the Servlet specification and RFC6265.
      */
-    class Mutable implements Iterable<HttpField>, HttpFields
+    class MutableHttpFields implements Mutable
```

Comment:
I wish this class could be hidden, and be out of the interface.

Reply:
yeah - I tried a number of things.... but then I actually do need to extend it sometimes, so the constructors are all protected, so that gives some limits